infrastructure/bin/deploy.sh

```diff
@@ -62,12 +62,15 @@ npm run production
 
 ### Indigenous Apprenticeship
 
-cd $ROOT_DIR/frontend/indigenousapprenticeship
+cd $ROOT_DIR
+npm install
 npm run codegen
 npm run intl-compile
-npm run production
+npm run build
 
-### Cleanup /frontend npm dependencies
 
+### Cleanup /frontend npm dependencies
```

Comment:
I tried to build this branch in Azure and it crashed out with the message `Missing script: "prune"`.  I think it's coming from this section.

Reply:
Oops, added `run` when I shouldn't have! Fixed in [fix deploy prune cmd](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5173/commits/0f464bf151a453c9b90887c4c13ad37e2f145bfe)